cypress/e2e/consortia/view-user-affiliations-accordion.cy.js

```diff
@@ -1,69 +1,54 @@
-import ConsortiumManager from '../../support/fragments/settings/consortium-manager/consortium-manager';
+import UsersSearchPane from '../../support/fragments/users/usersSearchPane';
+import UsersCard from '../../support/fragments/users/usersCard';
 
 describe('Consortia', () => {
-  let user;
+  let firstUser;
+  let secondUser;
 
   before(() => {
-    // cy.setTenant('college');
-    // // cy.getAdminToken();
+    cy.setTenant('consortium');
+    cy.loginAsConsortiumAdmin();
 
-    // cy.loginAsCollegeAdmin();
-
-    // cy.createTempUser([
-    //   permissions.uiSettingsOrdersCanViewAllSettings.gui,
-    //   permissions.uiOrdersCreate.gui,
-    //   permissions.uiOrdersEdit.gui,
-    // ], 'staff', 'staff').then((userProperties) => {
-    //   user = userProperties;
-    //   cy.login(user.username, user.password, {
-    //     path: TopMenu.ordersPath,
-    //     waiter: Orders.waitLoading,
-    //   });
-    // });
-    // cy.logout();
-    cy.setTenant('university');
-    // cy.getAdminToken();
-
-    cy.loginAsUniversityAdmin();
+    cy.createTempUser([]).then((userProperties) => {
+      firstUser = userProperties;
+    });
 
-    cy.createTempUser(
-      [
-        permissions.uiSettingsOrdersCanViewAllSettings.gui,
-        permissions.uiOrdersCreate.gui,
-        permissions.uiOrdersEdit.gui,
-        permissions.consortiaSettingsConsortiaAffiliationsView.gui,
-      ],
-      'staff',
-      'staff',
-    ).then((userProperties) => {
-      user = userProperties;
-      cy.login(user.username, user.password, {
-        path: TopMenu.ordersPath,
-        waiter: Orders.waitLoading,
+    cy.createTempUser([
+      permissions.consortiaSettingsConsortiaAffiliationsEdit.gui,
+      permissions.consortiaSettingsConsortiaAffiliationsView.gui,
+      permissions.uiUsersPermissionsView.gui,
+      permissions.uiUsersView.gui,
+    ]).then((secondUserProperties) => {
+      secondUser = secondUserProperties;
+      cy.login(secondUser.username, secondUser.password, {
+        path: TopMenu.usersPath,
+        waiter: Users.waitLoading,
       });
     });
   });
 
-  //   after(() => {
-  //     Users.deleteViaApi(user.userId);
-  //   });
-
-  it('Consortia', { tags: [TestType.criticalPath, devTeams.thunderjet] }, () => {
-    cy.visit(TopMenu.ordersPath);
-    ConsortiumManager.switchActiveAffiliation('Consortium');
-    cy.pause();
-    //   cy.logout();
+  after(() => {
+    cy.loginAsAdmin();
+    Users.deleteViaApi(firstUser.userId);
+    Users.deleteViaApi(secondUser.userId);
   });
+
+  it(
+    'C380505: View a users affiliations accordion in the third details pane (thunderjet)',
+    { tags: [TestType.smoke, devTeams.thunderjet] },
+    () => {
+      UsersSearchPane.searchByUsername(firstUser.username);
+      UsersSearchPane.selectUserFromList(firstUser.username);
+      UsersCard.varifyUserCardOpened();
+      UsersCard.expandAffiliationsAccordion();
+      UsersCard.verifyAffiliationsDetails('Consortium');
```

Comment:
Could you please add here a check that number of shown affiliations is equal to the number that was shown to the right of affiliations accordion header when it was collapsed?
And also a check that the same number is shown when accordion is collapsed again. As per test case

Reply:
added new method verifyAffiliationsQuantity()
and modified the verifyAffiliationsDetails() method